Automattic/jetpack
# Social: Add reel validation for Instagram connections

This change adds the validation for Instagram Reels, introducing a new logic for aspect ratio checks.


## Proposed changes:

* Added restrictions for instagram-business
* Added logic to check aspect ratio
* Updated messaging to mention `photo or video` on the Advanced plan.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






First test the messaging update.

* With both Classic editor and Gutenberg check the following:
  * With the Advanced plan on the site, the error with no media for instagram connections should mention videos 
  * Without the Advanced plan it should say only photos

Test the Instagram reel validation.

* Try uploading a video bigger than FHD, so wider than 1920px, it should result in an error.
* You can test the aspect ratio validation too by changing the values in `restrictions.js`
* Uploading videos that match the criteria should work.
